examples/topics/data_writing/google_sheet/.gitignore

```diff
@@ -0,0 +1,2 @@
+auth.json.dist
```

Comment:
it should be auth.json that is ignored not .dist - in fact .dist should be committed

Reply:
I needed to limit it slightly as GH blocked the commit due to security checks.